conf/app.routes

```diff
@@ -4,6 +4,8 @@ GET         /oracle/packaging-types                    uk.gov.hmrc.emcstferefere
 
 POST        /oracle/packaging-types                    uk.gov.hmrc.emcstfereferencedata.controllers.RetrievePackagingTypesController.show
 
+GET         /oracle/wine-operations                    uk.gov.hmrc.emcstfereferencedata.controllers.RetrieveWineOperationsController.showAllWineOperations
```

Comment:
Is it worth having an entry in the README?

Reply:
Updated in https://github.com/hmrc/emcs-tfe-reference-data/pull/40/commits/33ffbc444da83d429f4bdda6b1955b7726bc9671